SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -171,6 +173,13 @@ void USpatialReceiver::LeaveCriticalSection()
 void USpatialReceiver::OnAddEntity(const Worker_AddEntityOp& Op)
 {
 	UE_LOG(LogSpatialReceiver, Verbose, TEXT("AddEntity: %lld"), Op.entity_id);
+
+	if (EventTracer->IsEnabled())
```

Comment:
As per other comments, explicit `IsEnabled` check not required. If `FEventCreateEntity()` is expensive to call, `Trace_EventTracer_ShouldSampleEvent` should be called with a `Trace_Event` instance with its owning `span_id` set (which in this case should be the span ID attached to the `Worker_Op` which contained this `Worker_AddEntityOp` instance.